contributor-guide/modules/ROOT/pages/testing/continuous-integration.adoc

```diff
@@ -1,71 +1,94 @@
 = Continuous Integration
```

Comment:
Cirrus CI (https://cirrus-ci.org/) offers FreeBSD support. Considering its unique capability, could it be worth incorporating into the docs and the Boost continuous integration process?

Reply:
@fpelliccioni Probably a good question for @alandefreitas  or @anarthal ?

Reply:
Nothing against that. I just haven't seen it being used in other libraries yet.